packages/suite-build/configs/base.webpack.config.ts

```diff
@@ -197,6 +197,8 @@ const config: webpack.Configuration = {
               ]
             : []),
     ],
+    // For tiny-secp256k1
+    experiments: { asyncWebAssembly: true },
```

Comment:
is it required by tiny-secp256k1 or optional? if required then this should be added to all the projects which are using utxo-lib (connect, connect-iframe, connect-popup, connect-explorer)

i know that we build some other wasm files (cardano?), will this have effect on them as well?